### PR TITLE
Unauthenticated User Redirect to Static Content Page

### PR DESCRIPTION
**Pull request by bradbergeron-us**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

- *(Summarize the changes that have been made to the platform)*
- *(If bug, how to reproduce)*
- *(What is the solution, why is this the solution)*
- *(Which team do you work for, does your team own the maintenance of this component?)*
- *(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)*

Issue:  Users who are unauthenticated and go to the authenticated Enrollment Verification pages are seeing the default no enrollments page.

Expected Outcome:  Users who are unauthenticated and go to the authenticated Enrollment Verification pages should be redirected to the EV static landing page.

Additional Info:

Static EV landing page: https://www.va.gov/education/verify-school-enrollment/

Authenticated EV page: https://www.va.gov/education/verify-school-enrollment/enrollment-verifications/

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#0000
- *Link to ticket created in va.gov-team repo*
- *Link to previous change of the code/bug (if applicable)*
- *Link to epic if not included in ticket*

## Testing done

- *Describe what the old behavior was prior to the change*
- *Describe the steps required to verify your changes are working as expected*
- *Describe the tests completed and the results*
- *Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)*

## Screenshots
_Note: This field is mandatory for component work and UI changes (non-component work should NOT have screenshots)._

| | Before | After |
| --- | --- | --- |
| Mobile | | |
| Desktop | | |

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature
- [ ]  [Accessibility foundational testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by s-caso — APPROVED** [review]

**Review by nathanielbuck — APPROVED** [review]
These changes look good; the only thing I would do differently is to put the URL for the public content page ('/education/verify-school-enrollment') in the constants file, rather than hard-coding it in three places. Other than that, things look good.

File: src/applications/enrollment-verification/containers/EnrollmentVerificationPage.jsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export const EnrollmentVerificationPage = ({
   useEffect(
     () => {
       if (hasCheckedKeepAlive && !isLoggedIn) {
-        history.push('/');
+        window.location.href = '/education/verify-school-enrollment';
```
Comment (nathanielbuck) [inline src/applications/enrollment-verification/containers/EnrollmentVerificationPage.jsx:35]: I would put this URL in a constants file.

File: src/applications/enrollment-verification/containers/VerifyEnrollmentsErrorPage.jsx [hunk 2]
```diff
@@ -20,7 +23,7 @@ export const VerifyEnrollmentsErrorPage = ({
   useEffect(
     () => {
       if (hasCheckedKeepAlive && !loggedIn) {
-        history.push('/');
+        window.location.href = '/education/verify-school-enrollment';
```
Comment (nathanielbuck) [inline src/applications/enrollment-verification/containers/VerifyEnrollmentsErrorPage.jsx:26]: I would put this URL in a constants file too.

File: src/applications/enrollment-verification/containers/VerifyEnrollmentsPage.jsx [hunk 3]
```diff
@@ -55,7 +55,7 @@ export const VerifyEnrollmentsPage = ({
   useEffect(
     () => {
       if (hasCheckedKeepAlive && !isLoggedIn) {
-        history.push('/');
+        window.location.href = '/education/verify-school-enrollment';
```
Comment (nathanielbuck) [inline src/applications/enrollment-verification/containers/VerifyEnrollmentsPage.jsx:58]: I would put this URL in the constants file too.

File: src/applications/enrollment-verification/containers/VerifyEnrollmentsErrorPage.jsx (hunk 2 above)
Comment (bradbergeron-us) [reply src/applications/enrollment-verification/containers/VerifyEnrollmentsErrorPage.jsx]: Thanks for the suggestion.

**Pull request merged**